target/linux/ath79/dts/qca9556_mikrotik_rb-wapg-5hact2hnd.dts

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	compatible = "mikrotik,rb-wapg-5hact2hnd", "qca,qca9556";
+	model = "MikroTik RouterBOARD wAP G-5HacT2HnD (wAP AC)";
```

Comment:
The compatible doesn't really match the model name nicely. From model name, I'd have chosen
mikrotik,routerboard-wap-g-5hact2hnd

I do not insist on this one, but I think we should have this discussion here already when we start to add those devices.

Reply:
```root@OpenWrt:/# rbextract -p /dev/mtd1ro 
RouterBOARD wAP G-5HacT2HnD
```
"mikrotik,routerboard-wap-g-5hact2hnd" seems more reasonable then.